pym/bob/input.py

```diff
@@ -3083,6 +3084,11 @@ class RecipeSet:
             InfoOnce("managedLayers policy is not set. Only unmanaged layers are supported.",
                 help="See http://bob-build-tool.readthedocs.io/en/latest/manual/policies.html#managedlayers for more information.")
         ),
+        "urlScmSeparateDownload": (
+            "0.25.1.dev27",
+            InfoOnce("urlScmSeparateDownload policy is not set. Urlscm downloads resist in the workspace.",
```

Comment:
As this only affects downloaded files that are extracted, maybe the following would be more appropriate?

```
"urlScmSeparateDownload policy is not set. Extracted archives of the 'url' SCM are retained in the workspace."
```